consensus/obcpbft/obc-classic_test.go

```diff
@@ -25,13 +25,10 @@ import (
 	"github.com/spf13/viper"
 )
 
-func (op *obcClassic) getPBFTCore() *pbftCore {
-	return op.legacyGenericShim.pbft.pbftCore
-}
-
 func obcClassicHelper(id uint64, config *viper.Viper, stack consensus.Stack) pbftConsumer {
 	// It's not entirely obvious why the compiler likes the parent function, but not newObcClassic directly
-	return newObcClassic(id, config, stack)
+	config.Set("general.batchsize", 1)
+	return newObcBatch(id, config, stack)
 }
 
 func TestClassicNetwork(t *testing.T) {
```

Comment:
I can do this here, if we'd prefer.  I was thinking to wait to re-arrange the tests for another PR so as to minimize the diff, but I'm happy either way.